inheritance/src/test/java/inheritance/LibraryTest.java

```diff
@@ -18,16 +18,87 @@
     }
 
     @Test public void addReviewTEst() {
         restaurant.addReview(review);
-        assertEquals("Restaurant{name='Happy Teriyaki', rating=5, price_category='$$'}", restaurant.toString());
+        assertEquals("It should add review for the restaurant","Restaurant{name='Happy Teriyaki', rating=5.0, price_category='$$'}", restaurant.toString());
+
+    }
+
+    //Lab -07
+    @Test public void shopTest() {
+        Shop shop1 = new Shop("Macy","great place to hangout", "$$");
+        Shop shop2 = new Shop("Ross","great place to hangout", "$$");
+        Shop shop3 = new Shop("Dollar Tree","great place to hangout", "$$");
+        Shop shop4 = new Shop("Terrace","great place to hangout", "$$");
+
+        assertEquals("The shop constructor should create a new shop object","Shop{name='Macy', description='great place to hangout', dollarSigns='$$'}", shop1.toString());
+        assertEquals("The shop constructor should create a new shop object","Shop{name='Ross', description='great place to hangout', dollarSigns='$$'}", shop2.toString());
+        assertEquals("The shop constructor should create a new shop object","Shop{name='Dollar Tree', description='great place to hangout', dollarSigns='$$'}", shop3.toString());
+        assertEquals("The shop constructor should create a new shop object","Shop{name='Terrace', description='great place to hangout', dollarSigns='$$'}", shop4.toString());
+
+    }
+
+    @Test public void shopReviewTest() {
+        Shop shop = new Shop("Macy","great place to hangout", "$$");
+        Review review1 = new Review("Great!", "Bob", 4.3);
+        Review review2 = new Review("Great!", "John", 3.9);
+        Review review3 = new Review("Great!", "Stacy", 5.0);
+        Review review4 = new Review("Great!", "Karma", 4.0);
+        shop.addReview(review1);
+        shop.addReview(review2);
+        shop.addReview(review3);
+        shop.addReview(review4);
+
+        assertEquals("It should populate the review for the shop","Review{body='Great!', author='Bob', rating='4.3'}", shop.getReviews().get(0).toString());
+        assertEquals("It should populate the review for the shop","Review{body='Great!', author='John', rating='3.9'}", shop.getReviews().get(1).toString());
+        assertEquals("It should populate the review for the shop","Review{body='Great!', author='Stacy', rating='5.0'}", shop.getReviews().get(2).toString());
+        assertEquals("It should populate the review for the shop","Review{body='Great!', author='Karma', rating='4.0'}", shop.getReviews().get(3).toString());
+
+    }
+
+    @Test public void theaterTest() {
+        Theater theater1 = new Theater("Regal");
+        Theater theater2 = new Theater("Paccar");
+        Theater theater3 = new Theater("Cinerama");
+        Theater theater4 = new Theater("Big Picture");
+        assertEquals("The theater constructor should create a new theater object","Theater{name='Regal', allMovies=[  ]}", theater1.toString());
```

Comment:
this test fails, you have an extra space

Reply:
Fixed. Thanks. PR: https://github.com/jManij/java-fundamentals/pull/10